libcxx/test/std/utilities/memory/specialized.algorithms/specialized.construct/construct_at.pass.cpp

```diff
@@ -78,12 +86,70 @@ constexpr bool test() {
     a.deallocate(p, 2);
   }
 
+  // Test LWG3436, std::construct_at with array types
```

Comment:
IIRC Clang didn't support placement new for array in constant evaluation until Clang 20. (The ability was added via #104586.) So, should we skip "Clang 19 + constant evaluation" here?